packages/ui/src/ReactMarkdown.tsx

```diff
@@ -43,11 +44,13 @@ export const ReactMarkdown = React.memo(({content, navigateNote}: {content: stri
     );
 });
 
-const MarkdownCodeBlock: React.FC<any> = ({inline, className, children, ...props}) => {
+const MarkdownCodeBlock: (
+    props: React.JSX.IntrinsicElements['code'] & ExtraProps
+) => React.JSX.Element | string | null | undefined = ({node, className, children, ...props}) => {
```

Comment:
Weiter unten bei href ersetzt du ...props durch node?.properties  - sollte dass dann hier in Zeile 57 bei <code> auch so sein?

Reply:
Not sure, I replaced it in the <Link> because there was a type error. And as it was fine for <code> I kept it. But I can change it. It works both ways.

Reply:
Well, lets keep it this way then. I was merely wondering what was the reasoning :D

Reply:
I already changed it :p